docs/maintainers/concept-exercise-style-guide.md

```diff
@@ -0,0 +1,56 @@
+# Formatting and Style Guide for Concept Exercise Files
+
+This document records the current styleguide applicable to Concept exercise Markdown files, which refer to the files contained in `.docs` and `.meta`. Please follow them when implementing a Concept Exercise; maintainers should also reference the Style Guide when reviewing pull requests. When the relevant language also includes a style guide, those preferences will take precedence over the ones here.
+
+## Headers
+
+ - Use level 2 headers, e.g. `##`
+ - Do not use top-level headers, e.g. `#`
+ - Do not use leading whitespace in the file.
+
+
+## Empty Documents
+
+If a document is empty, i.e. there is no `after.md` content, then do not check one in.
+
+
+## Links
+
+Exercism prefers "indirect reference" links. They are defined at the bottom of the Markdown file and mapped to a reference slug, and referenced by that slug in the text.
```

Comment:
I think they're called [reference links](https://spec.commonmark.org/0.29/#reference-link). Maybe we can refer to the official docs:

```suggestion
Exercism prefers [reference links](https://spec.commonmark.org/0.29/#reference-link), which are defined at the bottom of the Markdown file, mapped to a reference slug, and referenced by that slug in the text.
```